CHANGELOG.md

```diff
@@ -1,3 +1,349 @@
+   `ChannelManager::splice_channel`.
+ * Various APIs have been updated to offer a native Rust async API. All
+   newly-async traits and structs have a `*Sync` variant which offers the
+   same API but with sync methods:
+   * `KVStore` has been made async. Note that `KVStore` methods are not
+     `async fn`, but rather write ordering is fixed when the methods return,
+     though write completion is async.
+   * `BumpTransactionEventHandler` is no backed by an async `WalletSource` (and
+     `Wallet`) or an async `CoinSelectionSource` and is now async. Sync versions
+     are in the new `sync` submodule (#3752).
+   * `OutputSweeper` is now backed by an async `KVStore` and
+     `ChangeDestinationSource` and is now async (#3819, #3734, #4131).
+   * `MonitorUpdatingPersisterAsync` and `ChainMonitor::new_async_beta` were
+     added for async `ChannelMonitor` persistence. Note that this feature is
+     still considered beta (#4063).
+ * An initial version of async payments is now supported. The current
+   implementation is specific to LDK and only LDK supports paying static
+   invoices. However, because BOLT 12 invoice requests will gracefully "upgrade"
+   to a non-static invoice when the recipient comes online, offers backed by
+   static invoices are expected to be payable by any BOLT 12 payer.
+   With an LDK-based LSP, often-offline clients should set
+   `UserConfig::hold_outbound_htlcs_at_next_hop` and call
+   `ChannelManager::set_paths_to_static_invoice_server`.
+   LDK-based LSPs wishing to support often-offline senders and recipients should
+   set `UserConfig::enable_htlc_hold`, support the existing "onion mesage
+   mailbox" feature (setting `intercept_messages_for_offline_peers` on
+   `OnionMessegner` and handling `Event::OnionMessageIntercepted`s), and handle
+   `Event::PersistStaticInvoice`s and `Event::StaticInvoiceRequested`s.
+ * Zero-Fee-Commitment channels are now supported in LDK. These channels remove
+   force-closure risk for feerate disagreements by using a fixed, zero fee on
+   presigned transactions, relying on anchor bumps instead. This only works with
+   LDK peers, and feature signaling may change in a future version of LDK,
+   breaking compatibility. This is negotiated automatically for
+   manually-accepted inbound channels and negotiated for outbound channels based
+   on `ChannelHandshakeConfig::negotiate_anchors_zero_fee_htlc_tx`.
+
+ * `Event::BumpTransaction` is now always generated even if the transaction has
+   sufficient fee. This allows you to manage transaction broadcasting more
+   granularly for anchor channels (#4001).
+ * The local key which receives non-HTLC-encumbered funds when the counterparty
+   force-closes a channel is now one of a static list of 1000 keys when using
+   `KeysManager` if `v2_remote_key_derivation` is set or after splicing (#4117).
+ * LSPS5 support was added, providing a push notification API for LSPS clients.
```

Comment:
Link to the LSPS5 PR is missing.

Reply:
I skipped links for features that had many PRs. In the case of LSPS5 there were at least 4/5 PRs with fixups so just left out links.

Reply:
Okay.